Makefile

```diff
@@ -85,15 +85,12 @@ helm-package: manifests
 	mkdir -p $(DIST_FOLDER)
 	@echo "Packaging Helm chart with version $(VERSION)..."
 	@cp helm/kmcp/Chart.yaml helm/kmcp/Chart.yaml.bak
-	@sed "s/^version: .*/version: $(VERSION)/" helm/kmcp/Chart.yaml.bak > helm/kmcp/Chart.yaml
+	@sed "s/^appVersion: .*/appVersion: \"$(VERSION)\"/" helm/kmcp/Chart.yaml.bak > helm/kmcp/Chart.yaml
```

Comment:
one last comment this doesn't work because `appVersion` doesn't exist in `chart.yaml` I'm thinking we can do this instead to make sure it gets added to the Chart.yaml file when the version is provided.